front_end/html/summarize_logs.html

```diff
@@ -0,0 +1,158 @@
+
+<form method="post" action="/summarize_logs">
+
+<div class="columns">
+  <div class="column is-half">
+    <h3>CodeBuddy Summarized Logs</h3>
```

Comment:
Please remove "CodeBuddy" from this. The name of the app is already in the nav bar.